cosmos/launchpad/construction_hash_test.go

```diff
@@ -0,0 +1,48 @@
+package launchpad
+
+import (
+	"context"
+	"encoding/base64"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+	"github.com/tendermint/cosmos-rosetta-gateway/rosetta"
+
+	"github.com/cosmos/cosmos-sdk/simapp"
+
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLaunchpad_ConstructionHash(t *testing.T) {
+	expectedHash := "6f22ea7620ebcb5078d244f06e88dd26906ba1685135bfc34f83fefdd653198a"
+
+	open, err := os.Open("./testdata/signed-tx.json")
```

Comment:
any reason for not using `ioutil.ReadFile` directly

Reply:
no, thanks! Updated!